content/setup/ctd.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Set Up a Compute-to-Data Environment
+description: Set Up a Compute-to-Data enviroment.
+---
+
+
+## Requirements
+
+At the time of writing, we need the following:
+First, create a folder with the following structure:
+
+- working [Barge](https://github.com/oceanprotocol/barge). For this setup, we will asume the Barge is installed in /ocean/barge/
```

Comment:
also note for later: it needs to be clarified what "the Barge is installed" means. Do you mean clone the repo into that folder structure? If so, say that